src/main/proto/workload.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
We could start packaging the Spiffe protos as maven packages during the release process. This allows users to specify specific versions through their dependency management instead of copy&pasting them inside their project.

Reply:
What you propose is to have a separate maven project that would have only the proto definitions ?